styles/style.css

```diff
@@ -95,8 +103,36 @@ p {
 
 /* Buttons */
 
-  /* INSERT BUTTON STYLES HERE */
+/* Change the background color when hovered over or when the button has the focus */
+
+button:hover {
+  background-color: #934423;
+}
+
+button:focus {
+  background-color: #934423;
+}
+
+.aButton {
```

Comment:
The two buttons share the similar styles which can be shared. You can have a generic button class which is used in those buttons, and have 2 classes to differentiate between the orange and white ones.